src/components/settings/SettingsAppearanceTab/CustomThemeModal.jsx

```diff
@@ -0,0 +1,137 @@
+  const invalidMessage = 'Must be hexidecimal starting with # (Ex: #1e0b57)';
+  const validHex = new RegExp('^#([A-Fa-f0-9]{6})$');
+
+  function validateColor(input, setter) {
+    if (!input.match(validHex)) {
+      setter(false);
+    } else {
+      setter(true);
+    }
+  }
+
+  function setCustom() {
+    const CUSTOM_THEME = {
+      title: 'Custom Theme',
+      button: (button === '' ? customColors?.button : button),
+      banner: (banner === '' ? customColors?.banner : banner),
+      accent: (accent === '' ? customColors?.accent : accent),
+    };
+    setTheme([CUSTOM_THEME, CUSTOM_THEME]);
+    close();
+  }
+
+  return (
+    <ModalDialog
+      title="My dialog"
+      isOpen={isOpen}
+      onClose={close}
+      size="md"
+      hasCloseButton
+      isFullscreenOnMobile
+    >
+      <ModalDialog.Header>
+        <ModalDialog.Title>
+          Custom Theme
+        </ModalDialog.Title>
+        <p className="mt-3">{headerText}</p>
+      </ModalDialog.Header>
+
+      <ModalDialog.Body>
+        <Form>
+          <Form.Group controlId="custom-button">
+            <Form.Control
+              floatingLabel="Button color"
+              isInvalid={!buttonValid}
+              onChange={(e) => {
+                setButton(e.target.value);
+                validateColor(e.target.value, setButtonValid);
+              }}
+              defaultValue={customColors?.button}
+            />
+            {!buttonValid && (<Form.Control.Feedback type="invalid">{invalidMessage}</Form.Control.Feedback>)}
+          </Form.Group>
+          <Form.Group controlId="custom-banner">
+            <Form.Control
+              floatingLabel="Banner color"
+              isInvalid={!bannerValid}
+              onChange={(e) => {
+                setBanner(e.target.value);
+                validateColor(e.target.value, setBannerValid);
+              }}
+              defaultValue={customColors?.banner}
+            />
+            {!bannerValid && (<Form.Control.Feedback type="invalid">{invalidMessage}</Form.Control.Feedback>)}
+          </Form.Group>
+          <Form.Group controlId="custom-accent">
+            <Form.Control
+              floatingLabel="Accent color"
+              isInvalid={!accentValid}
+              onChange={(e) => {
+                setAccent(e.target.value);
+                validateColor(e.target.value, setAccentValid);
+              }}
+              defaultValue={customColors?.accent}
+            />
+            {!accentValid && (<Form.Control.Feedback type="invalid">{invalidMessage}</Form.Control.Feedback>)}
+          </Form.Group>
+        </Form>
+        <p className="mt-4 mb-1 d-flex x-small">
+          <Icon src={InfoOutline} className="mr-2" /> {infoText}
+          <Hyperlink
+            destination={HELP_CENTER_BRANDING_LINK}
+            style={{ display: 'contents' }}
```

Comment:
nit/curious: why was this style override necessary? just wondering if there's anything needed to be improved on in the `Hyperlink` component in Paragon.

Reply:
Because we have the icon and text as two containers in a flexbox, I want the hyperlink to be considered the same part of the text as the regular words, not its own separate container in the flex box.